materialchart/src/main/java/app/priceguard/materialchart/Chart.kt

```diff
@@ -643,7 +645,10 @@ class Chart @JvmOverloads constructor(
                     circlePaint.setCirclePaint()
                     canvas.drawCircle(pointX, startY.value, circleSize.value / 2, circlePaint)
 
-                    val text = convertToText(data.y)
+                    val text =
+                        "${dataset?.xLabel ?: "x"} : ${convertTimeStampToDate(pointXData, dataset?.graphMode?: GraphMode.DAY)}," +
```

Comment:
This need to be reformatted.
`"${dataset?.xLabel ?: "x"} : ${convertTimeStampToDate(pointXData, dataset?.graphMode ?: GraphMode.DAY)},"`